src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SearchPageTelemetryEvent.cs

```diff
@@ -29,9 +30,27 @@ public class SearchPageTelemetryEvent : TelemetryEvent
             base["PageIndex"] = pageIndex;
             base["ResultCount"] = resultCount;
             base["Duration"] = duration.TotalSeconds;
-            base["IndividualSourceDurations"] = JsonConvert.SerializeObject(sourceTimings.Select(e => e.TotalSeconds));
+            base["IndividualSourceDurations"] = ToJsonArray(sourceTimings);
             base["ResultsAggregationDuration"] = aggregationTime.TotalSeconds;
             base["LoadingStatus"] = loadingStatus.ToString();
         }
+
+        private string ToJsonArray(IEnumerable<TimeSpan> sourceTimings)
+        {
+            _stringBuilder.Clear();
```

Comment:
Just create a local and eliminate the field:

    var stringBuilder = new StringBuilder();

...and make the method static.